models/ssh_key_test.go

```diff
@@ -56,6 +56,27 @@ func Test_SSHParsePublicKey(t *testing.T) {
 	}
 }
 
+func Test_CheckPublicKeyString(t *testing.T) {
```

Comment:
Just as a side comment, you can actually skip assigning to a variable as well if you'd like.
https://github.com/ncw/rclone/blob/8e8b78d7e57fae885a30df4d57eaa1da8f03dbd5/backend/sftp/sftp_internal_test.go#L39-L51

Reply:
Thanks, simplified.